### DISCUSSION TITLE
Problem/Issue Tracking for an asset.

### DISCUSSION THREAD

**Discussion by drunyan0824** [snipe/snipe-it#12184] [Q&A]
I am broadcast engineer and the IT department of my station just started to use Snipe-IT to track equipment and other assets. My boss has tasked me with trying to find a way to keep track of problems and issues that we are having with specific equipment. For example if we keep having to reboot a server because of a persistent error that causes the server to keep crashing. 

My boss wants to keep the record of the problems/issues of a piece of equipment separate from the regular maintenance records for the same piece of equipment.  I have an idea of how I can hopefully do this in a relatively easy way.  When I view an asset, I have the option to add a new maintenance record for the asset. A create asset maintenance window open when I click on add maintenance. On the create asset maintenance form there is a drop down menu labeled Asset Maintenance Type. I am wondering if there would be a way for my to add Problem/Issue Tracking as an option on the Asset Maintenance Type drop down menu?

Is that a realistic possibility of something that I might be able to add myself?

**Comment by hjertis** [comment]
I'd also love to add/remove types of maintenance from the drop down menu. It would be a super neat feature to have.
